app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt

```diff
@@ -197,7 +197,7 @@ class PatientSummaryEffectHandler @AssistedInject constructor(
             )
 
```

Comment:
Can we correct the commit message, please?
We have mistakenly spelled `hasPatientMeasurementDataChangedSinceScreenCreated` to `sPatientMeasurementDataChangedSinceScreenCreated`

Reply:
Ooh, good catch.

Reply:
@msasikanth I can't see the change here. Can you ptal?